.gitignore

```diff
@@ -0,0 +1,3 @@
+__pycache__
+test
+.vscode
```

Comment:
Please delete this `.gitignore`, because I want to keep the directory as simple as possible.
This decision may be not popular one, and I'm sorry that bother you. Also, you can use the global gitignore.